include/gnc/config.hpp

```diff
@@ -27,6 +27,9 @@
  *  used with floating point values. */
```

Comment:
When is GNC_NO_ASSERT defined? I'm uncomfortable with HOOTLs suddenly failing because over time, numerical errors cause these assertions to fail. A quieter error would be better. Is that possible?

Reply:
Yeah, I agree, also in the attitude estimator, assertions that vectors are normalized will fail if the vector has NANs. I like the use of asserts to though, they might make debugging weird numerical issues a lot easier, maybe though there could be an option for printing a warning and setting output to NAN or something instead of crashing?

Reply:
I sort of assumed we would disable it eventually in FSW. Also, I share the same worries which is both why the tolerances are fairly loose (and can be made looser if we have issues) and NaN checks are done prior to asserts (in functions that expect NaNs at least).

Reply:
Using NaNs to signal errors works but it can almost give us just as much trouble as assertions because it's very difficult to track the origin of the problem.

Reply:
I do like the idea of printing something, although given our wide use of platforms including mex functions, teensies, desktop, etc. handling the different forms of IO turns into a decent amount of work. I'd be in favor of leaving it as is for now and if we run into issues we can look into better definitions of `GNC_ASSERT`?